### PR TITLE
VACMS-6573: Add ehr field to vamc system type.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
Relates to #6573

## Testing done
Visual / behat / script execution

## QA steps
 - [x] As an administrator, go to `admin/structure/types/manage/health_care_region_page/form-display` and visually verify `Electronic Health Records system` is disabled (will default to select list, b/c it is disabled - we will need to toggle it back to radios once enabled)
 - [x] As an administrator, go to `admin/structure/types/manage/health_care_region_page/display` and visually verify `Electronic Health Records system` is disabled
 - [x] Go to terminal on tugboat and run `drush scr ./scripts/content/VACMS-6573-set-default-ehr-on-systems.php`
 - [x] Enable form visibility for `Electronic Health Records system` field
 - [x] Go to `node/15038/edit` and visually verify value is set to `Cerner`
 - [x] Go to `node/15082/edit` and visually verify value is set to `VistA`
 - [x] Go to `node/15083/edit` and visually verify value is set to `VistA`
 - [x] Go to `node/15084/edit` and visually verify value is set to `VistA`

### CODE REVIEW

**Review by kevwalsh — APPROVED** [review]
QA pass, looks good.

File: scripts/content/VACMS-6573-set-default-ehr-on-systems.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    $node->get('moderation_state')->setValue($current_moderation_status);
+    // Make this change a new revision.
+    $node->setNewRevision(TRUE);
+    // Set revision author to uid 1317 (CMS Migrator user).
+    $node->setRevisionUserId(1317);
+    $node->setChangedTime(time());
+    $node->isDefaultRevision(TRUE);
+    $node->setRevisionCreationTime(time());
+    // Set our default EHR to vista.
+    $ehr_val = $node->id() !== '15038' ? 'vista' : 'cerner';
```
Comment (swirtSJW) [inline scripts/content/VACMS-6573-set-default-ehr-on-systems.php:31]: Just noting that I validated 15038 is VA Spokane.  The only system that is using Cerner.

**Review by swirtSJW — APPROVED** [review]

**Comment by swirtSJW** [comment]
image

**Comment by swirtSJW** [comment]
Post-deploy steps needed.
- [ ] On prod run   `drush scr ./scripts/content/VACMS-6573-set-default-ehr-on-systems.php`
- [ ] Post a comment on this PR with the terminal output from running the command.

**Comment by ndouglas** [comment]
```bash
[cms@ip-10-247-32-171 cms]$ drush scr ./scripts/content/VACMS-6573-set-default-ehr-on-systems.php
Set default EHR on in '139' nodes.
```

**Pull request merged**